.github/workflows/wheel.yaml

```diff
@@ -27,11 +32,23 @@ jobs:
         with:
           python-version: '3.9'
 
+      - uses: docker/setup-qemu-action@v1
+        if: ${{ matrix.arch == 'aarch64' }}
+        name: Set up QEMU
+
       - name: Install cibuildwheel
         run: |
           python -m pip install cibuildwheel==1.8.0 wheel
 
+      - name: Build wheel for aarch64
+        if: ${{ matrix.arch == 'aarch64' }}
+        env:
+          CIBW_ENVIRONMENT: 'DISABLE_NUMCODECS_AVX2=1 DISABLE_NUMCODECS_SSE2=1'
```

Comment:
Could these have also be set in the setup.py rather than `pass`ing? Just wondering if having a single place to look would prevent future issues.